### PR TITLE
Added page extraction module

### PR DESCRIPTION
**Pull request by AuCson**
Hi,

Thanks for the nice work! I added some lines of codes to support extracting pages from the PDF given the page range, which I find to be quite useful. Currently the function is merged into the "rotate page" module.

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 APPNAME = 'pypdfbuilder'
-APPVERSION = '0.9'
+APPVERSION = '0.9.0_fork_YXVjc29u'
```
Comment (teschmitt) [inline settings.py]: Could you please change this to 0.9.1?

**Review by teschmitt — CHANGES_REQUESTED** [review]
Wow I never thought this would be useful to someone … thank you so much for your contribution! I have one small change request: could you change the `APPVERSION` to `0.9.1`? But other than that it's looking good.

File: settings.py (hunk 1 above)
Comment (AuCson) [reply settings.py]: Sure, I changed that in my fork

**Comment by teschmitt** [comment]
Awesome, thanks again!

**Pull request merged**